### PR TITLE
update to fly@0.7.0 new API

### PR DESCRIPTION
**Pull request by ghost**
we still need to add tests. I will try to get this done later this week too.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -2,13 +2,8 @@ const less = require("less")
 const assign = require("object-assign")
 
 module.exports = function () {
-  this.filter("less", (source, options) => {
-    options = assign({}, {
-      compress: false,
-      paths: []
```
Comment (ghost) [inline index.js:8]: If `@import` is still not working, we can add this bit again.

**Comment by watilde** [comment]
@bucaran Can I take over this PR?

**Comment by ghost** [comment]
@watilde Of course and thanks.

**Comment by watilde** [comment]
I updated some code for fixing test case and rebased.

**Comment by watilde** [comment]
Seems ok! After merge this PR, I'll publish as minor updating.

**Comment by watilde** [comment]
Done https://github.com/watilde/fly-less/releases/tag/0.3.0

**Comment by ghost** [comment]
Cool. Don't you think it would be better to test the plugins _without_ Fly? That's what I do [here](https://github.com/bucaran/fly-babel/blob/master/test/index.js) and it works OK.

**Pull request merged**